samples/05-flexbox-responsive-page/flex-page.css

```diff
@@ -0,0 +1,50 @@
+/* https://css-tricks.com/snippets/css/a-guide-to-flexbox/ */
```

Comment:
Probably needs a license hint or better ref to the author

Reply:
Yep I agree, that sounds reasonable